src/App.tsx

```diff
@@ -179,6 +183,7 @@ const resources = [
   ...chargingProfilesResources,
   ...messageInfosResources,
   ...variableMonitoringsResources,
+  ...VariableAttributesResources,
```

Comment:
(nitpick) - looks like we have been naming the resources with lower case first letter 🙏

Reply:
Valid point and fixed